source/guide_maildrop-autoresponder.rst

```diff
@@ -0,0 +1,260 @@
+
+
+**This is the place to configure the autoresponder.** We`ll set a reference to a ``reply-message.txt`` (or any other named)-file somewhere on your uberspace. ``mailbot`` will take the content of the specified file and use it as text for the reply-mail without further adjustments.
+  
+.. note:: This method is simple and efficient if you use it for yourself. But if you're planning to use this solution to handle multiple users with their own mailboxes and you aren't willing to give everyone access to the server by SSH, SFTP etc. to modify their response-messages head over to `One more thing`_ for an additional method.
+
+``mailbot`` requires some additional parameters:
+
+  * ``-t $HOME/autoresponder/autoresponse.txt``: Read the text for the autoresponse from the referenced file. 
+  * ``-A 'From: $FROM'``: Adds a header to the response. In our case, we'll set the ``From:`` header in the autogenerated response.
+  * ``-N``: Disables quoting the original message
+  * ``-d $HOME/autoresponder/bouncedb``: Creates a small database ``bouncedb`` in the specified path, that keeps track of senders' E-mail addresses, and prevent duplicate autoresponses going to the same address. Another autoresponse to the same address will not be mailed until at least the amount of time specified by the -D option has elapsed.
+  * ``-D 3``: Do not send duplicate autoresponses (see the -d flag) for at least ``3`` days (default: 1 day).
+  * ``/var/qmail/bin/qmail-inject -f ''``: utility to send the mail
+  
+.. note:: Refer to the `mailbot documentation`_ for more detailed informations how to use flags.
+  
+We have to make sure that all referenced directories and files exist, otherwise mailbot will abort immediately. Since we referred the ``autoresponse.txt`` file in the configuration above we 'll create it.
+
+Finally, add this last block to your ``~/autoresponder/.autoreply-filter``:
+
+.. code-block:: ini
+
+	# Receive the mail (the original mail goes into your INBOX)
+	to "$MAILDIR"
+	
+This line makes sure that the email is delivered to your ``$MAILDIR`` in any case, no matter which filters & rules were triggered before.
+
+
+.. warning:: Attention: The configuration must only be readable by the user who executes the maildrop command, **otherwise maildrop aborts immediately without delivering the e-mail**. For this reason, the access rights must be set after creation:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ chmod 600 ~/autoresponder/.autoreply-filter
+ 
+Create the qmail
+----------------
+
+Use your favourite editor to create ``~/.qmail-email`` with the following content (including the pipe):
+
+.. code-block:: ini
+
+	|maildrop $HOME/autoresponder/.autoreply-filter
+	
+All incoming mails will now be forwarded to ``maildrop``, which executes the filterfile ``~/autoresponder/.autoreply-filter`` that we will create in the next step. 
+
+.. warning::  Make sure that you use UNIX Line-endings_ in this file, **otherwise it won't work.** 
```

Comment:
maybe explain what UNIX line endings are or just include a step to send the file though `dos2unix` by default

Reply:
I've allready done this. The `UNIX Line-endings_` links to the section `Common isssues` below where the dos2unix prompt is given as a example to solve this issue.